ONSdigital/response-operations-ui
# Fix the resend verification email link for respondents that have changed email addresses

# Motivation and Context
Service desk call TASK0241375

A bug exists where if a respondent has requested a change of email address then requested the verification email to be resent, the link was not working and redirected to a 'URL not found' page.

# How to test?
- Register a new respondent without verifying the account and check that you can resend the verification email
- Change the respondent's email address
- Click the link to resend the verification email
- The link should now work and inform you that the email is being sent to their new address
